### ISSUE TITLE
Uncertainty Quantification (UQ) for PyTorch backend

### ISSUE THREAD

**Issue by jamesdolezal** [jamesdolezal/slideflow#147]
## Feature
Currently, the PyTorch backend does not have the ability to enable dropout via inference, which enables UQ & related features.

As dropout layers are added with hidden layers, part of the issue is that adding hidden layers works slightly differently in Torch & Tensorflow packends. Torch does not include the include_top functionality that Tensorflow does when loading pretrained models, which makes adding hidden layers less straightforward. In order to ensure identical performance between backends, the implementation of hidden layers needs to be verified for both backends. Once hidden layers are reliably identical between backends and work for all models, adding dropout layers that are active during inference will be trivial.
